umh.docs.umh.app/content/en/docs/getstarted/calculatekpiwithanalytics.md

```diff
@@ -0,0 +1,364 @@
+3. Set the topic to `umh/v1/printingCo/lisbon/hall-a/speedmaster106/_analytics/shift/add`.
+4. Drag a function node from the palette to the flow.
+5. Double-click the function node and set the following code:
+```javascript
+const startTime = Date.now();
+// This shift will be 8 hours
+const endTime = startTime + 28800000
+
+msg.payload = {
+    "startTime": startTime,
+    "endTime": endTime
+}
+return msg;
+```
+6. Drag an mqtt out node from the palette to the flow.
+7. Configure the mqtt node to use `united-manufacturing-hub-mqtt` as the Server, you can leave the topic empty as we have already set it in the inject node.
+8. Connect the inject node to the function node and the function node to the mqtt node.
+9. Deploy the flow.
+
+### Step 3: Creating the work-order/create function
+
+Now that we have a product-type and a shift, we can create a work-order. In this example we will create a work-order for 7500 "otto-poster" prints.
+Each work-order has an work order id, which is unique per asset. In this example we will use the work order id "#1247".
+This order id will come from your ERP system, but for this example we will use an inject node.
+
+1. Drag an inject node from the palette to the flow.
+2. Double-click the inject node and set the payload to:
+```json
+{
+  "externalWorkOrderId": "#1247",
+  "product": {
+    "externalProductId": "otto-poster"
+  },
+  "quantity": 7500
+}
+```
+3. Set the topic to `umh/v1/printingCo/lisbon/hall-a/speedmaster106/_analytics/work-order/create`.
+4. Drag an mqtt out node from the palette to the flow.
+5. Configure the mqtt node to use `united-manufacturing-hub-mqtt` as the Server, you can leave the topic empty as we have already set it in the inject node.
+6. Connect the inject node to the mqtt node.
+7. Deploy the flow.
+
+### Step 4: Creating the work-order/start function
+
+We are ready to start the work-order. This is done by creating a work-order/start event. In this example we will start the work-order "#1247", which we created in the previous step.
+
+1. Drag an inject node from the palette to the flow.
+2. Double-click the inject node and set the payload to:
+```json
+{
+  "externalWorkOrderId": "#1247"
+}
+```
+3. Set the topic to `umh/v1/printingCo/lisbon/hall-a/speedmaster106/_analytics/work-order/start`.
+4. Drag a function node from the palette to the flow.
+5. Double-click the function node and set the following code:
+```javascript
+const startTime = Date.now();
+
+msg.payload["startTime"] = startTime
+return msg;
+```
+6. Drag an mqtt out node from the palette to the flow.
+7. Configure the mqtt node to use `united-manufacturing-hub-mqtt` as the Server, you can leave the topic empty as we have already set it in the inject node.
+8. Connect the inject node to the function node and the function node to the mqtt node.
+9. Deploy the flow.
+
+
+### Step 5: Creating the state machine
+
+This step is quite complex. We will create a state machine to simulate the printing process.
```

Comment:
This looks more complicated than it is. How about that we start with two inject nodes "start machine" / "stop machine" just to show how states work. Then maybe one additional one for "produce product" (and note that one could now theoretically derive the machine state from whenever a product was produced).

"Normally you get the machine state directly from the machine. In the first step, we will simply inject a "machine running", and "machine stopped" with inject node. As a more advanced step, we will build a simulator in Node-RED.